web/crux/src/app/registry/registry.mapper.ts

```diff
@@ -75,6 +75,12 @@ export default class RegistryMapper {
               token: registry.token,
               imageNamePrefix: registry.imageNamePrefix,
             },
+      unchecked:
```

Comment:
It could be, IMHO

```js
unchecked: registry.type === RegistryTypeEnum.unchecked && { url: registry.url },
```

Reply:
I tried to follow the format used above, also the formatter splits it into multiple lines.

Reply:
i prefer the current one with the explicit null